tests/ship_streamer_test.py

```diff
@@ -137,7 +137,7 @@ def getLatestSnapshot(nodeId):
     nonProdNode.waitForTransBlockIfNeeded(trans, True, exitOnError=True)
     for account in accounts:
         Print(f"Transfer funds {transferAmount} from account {cluster.eosioAccount.name} to {account.name}")
-        nonProdNode.transferFunds(cluster.eosioAccount, account, transferAmount, "test transfer", waitForTransBlock=False)
+        trans=nonProdNode.transferFunds(cluster.eosioAccount, account, transferAmount, "test transfer", waitForTransBlock=False)
```

Comment:
This clobbers `trans` for each transfer, thus only waiting on the last one. I guess no material problem?

Reply:
That is the idea. Only wait on the last one. Although, you could argue that it would be best to verify they all make it, but this is certainly faster than that.

Reply:
> Although, you could argue that it would be best to verify they all make it,

yeah.. that's kind of where I was thinking. But it seems unnecessary for this test